src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -13,51 +14,90 @@
 import com.bumptech.glide.request.transition.Transition;
 
 import com.microsoft.maui.ImageLoaderCallback;
+import com.microsoft.maui.PlatformAppCompatImageView;
+import com.microsoft.maui.PlatformInterop;
 
 public class MauiCustomViewTarget extends CustomViewTarget<ImageView, Drawable> {
+    private static final String TAG = "MauiCustomViewTarget";
+    private static final boolean IS_VERBOSE_LOGGABLE = Log.isLoggable(TAG, Log.VERBOSE);
```

Comment:
I like the logging you've added throughout, but should we have a single tag, `Log.isLoggable("Glide", Log.VERBOSE)` for now? And not create multiple tags? Maybe also create a `GlideLogging` class to centralize it?